test/terra/backends/simulator_test_case.py

```diff
@@ -66,12 +74,39 @@ def _method_device(methods):
     if not methods:
         methods = AerSimulator().available_methods()
     available_devices = AerSimulator().available_devices()
+    #add special test device for cuStateVec if available
+    cuStateVec = check_cuStateVec(available_devices)
+
     gpu_methods = ['statevector', 'density_matrix', 'unitary']
     data_args = []
     for method in methods:
         if method in gpu_methods:
             for device in available_devices:
                 data_args.append((method, device))
+            #add test cases for cuStateVec if available using special device = 'GPU_cuStateVec'
+            #'GPU_cuStateVec' is used only inside tests not available in Aer
+            #and this is converted to "device='GPU'" and option "cuStateVec_enalbe = True" is added
+            if cuStateVec:
+                data_args.append((method, 'GPU_cuStateVec'))
```

Comment:
@chriseclectic could you review this change? This is a hack to minimize changes of tests for tests of `cuStateVec` option. `cuStateVec` is an option only for `device=GPU`. Current annotator `supported_methods()` requires tests to take two argument `method` and `device`. Adding new option `cuStateVec_enable` to all the tests is not productive, I believe.